content/advanced/fine-grained-reactivity.mdx

```diff
@@ -0,0 +1,222 @@
+# Understanding Fine-Grained Reactivity
+
+Reactivity is a programming pattern that lets you set up behavior that "reacts" to data changes automatically.
+
+For example, you might make a shopping cart with a label that displays the total cost of the items. With reactivity, you can tell that label to automatically update whenever an item gets added or removed.
+
+This way, you only have to manage the data. Once you wire up the UI elements to the data, the UI updates are handled for you.
+
+To understand how Solid handles fine-grained reactivity, a basic understanding of reactive signals and effects is required. A deeper understanding of reactivity can help you optimize your code, debug issues, and build your own extensions to the reactive system.
```

Comment:
Maybe first introduce that "fine grained reactivity" is an approach where only minimally necessary code needs run in order to achieve the behavior we want.